SpinCore_pp/config_parser_fn.py

```diff
@@ -124,21 +124,15 @@ class configuration(object):
             None,
             "the range over which the dip lock will be performed",
         ),
-        "FIR_rep": (
-            float,
-            "odnp_params",
-            None,
-            "fast inversion recovery repetition delay (should be around 2 x T1",
-        ),
-        "adc_offset": (int, "acq_params", None, "analog to DC conversion factor"),
```

Comment:
we deleted this b/c Weiss recommends a repetition delay that's calculated from the same parameters as the vdlist